apps/app/src/views/routes/Settings/components/utils/UserFragment.tsx

```diff
@@ -0,0 +1,30 @@
+import { graphql } from 'apps/app/src/gql';
+export const userFragment = graphql(/* GraphQL */ `
```

Comment:
This should be defined and in the specific files its being used and only specify the specific fields needed. It should also be named something like `<Component>UserFragment` (e.g. `EditRolesActionUserFragment`). Fragments need to be globally uniquely named

Reply:
done

Reply:
removed